contrib/python/api/skydive/rest/client.py

```diff
@@ -167,3 +167,33 @@ def edgerule_list(self):
     def edgerule_delete(self, rule_id):
         path = "/api/edgerule/%s" % rule_id
         self.request(path, method="DELETE")
+
+    def injection_create(self, srcQuery="", dstQuery="", type="icmp4",
+                         payload="", interval=1000, srcIP="",
+                         dstIP="", srcMAC="", dstMAC="",
+                         count=1, ICMPID=0, srcPort=0, dstPort=0):
+
+        path = "/api/injectpacket"
```

Comment:
this url prefix is shared by all added functions - please define it as a const outside this function blockL

```
INJECT_PATH = "/api/injectpacket"
```